deploy/test-environments/README.md

```diff
@@ -0,0 +1,96 @@
+# Test Environments Deployment
+
+**Motivation**
+To provide an easy and deterministic way to set up the latest cloud environment, ensuring proper monitoring and usability
+
+
+**Prerequisite**
+
+This project utilizes AWS and Elastic Cloud accounts. To ensure proper deployment and usage, it is essential to obtain appropriate licenses in compliance with the licensing terms and conditions provided by the respective service providers.
+
+Follow the [prerequisites](/README.md#prerequisites) chapter of our main README.
+
+## Environment Variables
+
+To successfully deploy the environment, ensure that the following variables are provided as deployment parameters or exported as environment variables:
+
+```bash
+export TF_VAR_ec_api_key={TOKEN} # <-- should be replaced by Elastic Cloud TOKEN
```

Comment:
Can you add the link somewhere (https://cloud.elastic.co/deployment-features/keys)

Reply:
readme file is updated